sssom/cli.py

```diff
@@ -434,8 +442,19 @@ def merge(inputs: str, output: TextIO, reconcile: bool = True):
     """  # noqa: DAR101
     msdfs = [read_sssom_table(i) for i in inputs]
     merged_msdf = merge_msdf(*msdfs, reconcile=reconcile)
-    # Export MappingSetDataFrame into a TSV
-    write_table(merged_msdf, output)
+    # Reconcile prefixes if needed.
+    if reconcile_prefixes:
```

Comment:
What happens if msdf1 says:

```
ICD10: ABC
```

and msdf2 says:

```
ICD10: DEF
```

Wouldnt that cause a clash just before resolution?

Maybe we should create a new client method, `reconcile_prefixes()` that takes as an input a sssom table and applies the reconciled prefixes. Sorry about that, I didn't really think this through.. That way the client is responsible for reconciling prefixes before running the merge.. So you don't need the option in `merge` at all, just add to a new, stand alone method that reads a sssom table, reconciles, and writes again.

Reply:
If there is a `reconcile_prefix.yaml` file, that should take care of such clashes. But in the even of the absence, I think the prefix from `msdf1` supersedes the one in `msdf2`.